endpoints.md

```diff
@@ -45,6 +45,13 @@
 * [/api/statistics](statistics.md)
 * [/api/tools/itemrequests](item-requests.md)
 
+## Actuator endpoints
+The following endpoints are implemented using [Spring Boot Actuator](https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.enabling) and are enabled by default:
+* /actuator/info 
+* /actuator/health
+
+More details [here](actuator.md)
```

Comment:
Linking text to the word "here" is not recommended as it's really bad for accessibility.  Let's just change this to say:  

`For more details see the [/actuator endpoint](actuator.md) documentation`